src/twisted/python/compat.py

```diff
@@ -822,6 +822,32 @@ def _bytesRepr(bytestring):
         return 'b' + repr(bytestring)
 
 
+
+def toBytes(s, encoding="ascii", errors="strict"):
+    """
+    Convert L{unicode} string to L{bytes}.
+
+    @param s: The string to convert.
+    @param encoding: The encoding to pass to L{str.encode}.
```

Comment:
Why no `@type` for the parameters?  Especially since s can be None as well as bytes or unicode.